install/terraform/modules/gke/cluster.tf

```diff
@@ -146,7 +146,7 @@ resource "google_container_cluster" "primary" {
 }
 
 resource "google_compute_firewall" "default" {
-  name    = "game-server-firewall-firewall-${local.name}"
+  name    = length(var.firewallName) == 0 ? "game-server-firewall-${local.name}" : var.firewallName
```

Comment:
Why this way instead of setting the default name in variables.tf?

Reply:
Ah! Excellent question. Had to do it this way, because you can't use use [Terraform string templates](https://www.terraform.io/docs/configuration/expressions.html#string-templates) in the default values of `var` declarations in Terraform.

Which took me a while to work out how to resolve!

Reply:
And we don't want the firewall name to be a fixed string by default?

Reply:
The default had the cluster name in it, so I figured it made sense to keep it that way.

(Probably to make it relatively unique, so it didn't collide with anything?)

Reply:
@aLekSer can speak to this, because you wrote the original version, from memory?

Reply:
Yes, that's true, in the other way we would have only one firewall, and creating the second cluster would fail.